src/text_shaper/font.h

```diff
@@ -119,6 +119,7 @@ struct font_description
 #if _WIN32
```

Comment:
```suggestion
#if defined(_WIN32)
```

At least that would be more consistent to the rest of the ifdef's to _WIN32 :)

Reply:
Sure,